chapters.md

```diff
@@ -17,8 +17,65 @@ As an example, consider a `Parent ChapterAtom Element` that has its `ChapterFlag
 
 Three `Edition Flags` are defined to describe the bevahior of the `EditionEntry Element`: `EditionFlagHidden`, `EditionFlagDefault` and `EditionFlagOrdered`.
 
+
+#### `EditionFlagHidden`
+
 The `EditionFlagHidden` Flag behaves similarly to the `ChapterFlagHidden` Flag: if `EditionFlagHidden` is set to `true` then its `Child ChapterAtoms Elements` MUST also be interpretted as if their `ChapterFlagHidden` is also set to `true`, regardless of their own `ChapterFlagHidden` flags. If the `EditionFlagHidden` is toggled by a `Control Track` to `false` then the `ChapterFlagHidden` Flags of the `Child ChapterAtoms Elements` SHALL determine if the `ChapterAtom` is hidden or not.
 
+
+#### `EditionFlagDefault`
+
+It is recommended that only one Edition's `EditionFlagDefault` Flag is set to `true`. If more then one `Default Edition` are present, the first `Default Edition` MUST be used.
+If an `EditionFlagDefault` Flag is set to `true` then MUST be played this Edition. When all `EditionFlagDefault` Flags are set to `false` then the first Edition MUST be used. 
+
+
+#### `EditionFlagOrdered`
+ 
+The `EditionFlagOrdered` Flag is a very powerful and most important feature.
```

Comment:
This seems to be more of an opinion and maybe not in scope for the specification? If it does stay, I suggest a re-phrasing such as "The `EditionFlagOrdered` Flag is a significant feature."